pkgs/by-name/cr/cros-ectool/package.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, lib }:
+
+pkgs.stdenv.mkDerivation rec {
+  name = "cros-ectool";
+  nativeBuildInputs = with pkgs; [ cmake ninja pkg-config libusb libftdi1 ];
+  src = pkgs.fetchFromGitLab {
+    domain = "gitlab.howett.net";
+    owner = "DHowett";
+    repo = "ectool";
+    rev = "39d64fb0e79e874cfe9877af69158fc2520b1a80";
+    hash = "sha256-SHRnyqicFlviBDu3aH+uKVUstVxpIhZV6JSuZOgOwXU=";
+  };
```

Comment:
This is a reupload of the upstream source. Could it be built from the upstream source instead, like [in Terra](https://github.com/terrapkg/packages/blob/aa7dcf65e32e2ec76f286dc2623eb8c7655c823f/anda/tools/chromium-ectool/chromium-ectool.spec#L11)?

Reply:
Maybe. Idk what commands are being run in Terra though.